utils/index.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -exo pipefail
+
+setup(){
+    # Generate a uuid
+    export UUID=${UUID:-$(uuidgen)}
+    # Elasticsearch Config
+    export ES_SERVER=$ES_SERVER
+    export ES_INDEX=$ES_INDEX
+    if [[ -z $PROW_JOB_ID ]]; then
+        export dag_id=${AIRFLOW_CTX_DAG_ID}
+        export execution_date=${AIRFLOW_CTX_EXECUTION_DATE}
+        export dag_run_id=${AIRFLOW_CTX_DAG_RUN_ID}
+        export ci="AIRFLOW"
+        printenv
+        # Get Airflow URL
+        export airflow_base_url="http://$(kubectl get route/airflow -n airflow -o jsonpath='{.spec.host}')"
+        # Setup Kubeconfig
+        export KUBECONFIG=/home/airflow/auth/config
+        curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
+        export PATH=$PATH:/home/airflow/.local/bin:$(pwd)
+        # ReleaseStream is piped in via environment variables
+        release_stream=${RELEASE_STREAM}
+    else
+        export ci="PROW"
+        export prow_base_url="https://prow.ci.openshift.org/view/gs/origin-ci-test/logs"
+    fi
+    # Get OpenShift cluster details
+    cluster_name=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}') || echo "Cluster Install Failed"
```

Comment:
Is there a way to be able to get this information from the metadata in ocp-commons? or the previously submitted es clusterMetadata data?

Reply:
Assuming by ocp-commons we are referring to go-commons here, actually the idea around having this implementation is to have a separate index which stores the data related to all our CI systems for each CI run.  And also go-commons is a GO library which is out of the scope for this implementation. Thank you!